README.md

```diff
@@ -1,4 +1,6 @@
-# Getting Started with Create React App
+# Math Magicians
+
+## Getting Started with Create React App
```

Comment:
- [ ] Please let's make our README file descriptive and professional. It is advisable to always make use of the [template](https://github.com/microverseinc/readme-template) provided by Microverse and don't forget to customize it to match your awesome project.